lighthouse-core/test/audits/long-tasks-test.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const LongTasks = require('../../audits/long-tasks.js');
+const createTestTrace = require('../create-test-trace.js');
+const devtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+
+const TASK_URL = 'https://pwa.rocks/';
+
+/* eslint-env jest */
+
+/**
+ * @param {Number} count
+ * @param {Number} duration
+ * @param {Boolean} withChildTasks
+ */
+function generateTraceWithLongTasks(
+  count,
+  duration = 200,
+  withChildTasks = false
+) {
+  const baseTs = 1000;
+  const traceTasks = [];
+  for (let i = 1; i <= count; i++) {
+    const ts = baseTs * i;
```

Comment:
looks like tasks start at 1s instead of 0, is that important?

Reply:
For this case, yes. I want to avoid any overlapping trace events so that no unintentional parent/child task relationships get formed since we need top level tasks for this audit. So starting at 1s avoids any of the added tasks overlapping with the required tasks defined in `CreateTestTrace()`.